apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/Options.kt

```diff
@@ -22,6 +23,10 @@ const val MODELS_OPERATION_BASED = "operationBased"
 @ApolloDeprecatedSince(v3_0_0)
 const val MODELS_COMPAT = "compat"
 
+const val ADD_TYPENAME_IF_FRAGMENTS = "ifFragments"
+const val ADD_TYPENAME_IF_POLYMORPHIC = "ifPolymorphic"
+const val ADD_TYPENAME_IF_ABSTRACT = "ifAbstract"
```

Comment:
Note: we do not check the API on apollo-compiler so this didn't appear in the api dump. Something we could do at some point